docs/api/rest_api/rest_api_reference/input/ez-user-users.raml

```diff
@@ -303,3 +303,50 @@ head:
                                 example: !include examples/user/users/user_id/roles/role_id/DELETE/RoleAssignmentList.json.example
                     401:
                         description: Error - the user is not authorized to delete this Content Type.
+        /segments:
+            get:
+                displayName: List User Segments
+                description: Load Segments for a given User.
+                headers:
+                    Accept:
+                        example:  |
+                            application/vnd.ibexa.api.UserSegmentList+xml
+                            application/vnd.ibexa.api.UserSegmentList+json
+                responses:
+                    200:
+                        body:
+                            application/vnd.ibexa.api.UserSegmentList+json:
+                                type: UserSegmentList
+                                example: !include examples/user/users/user_id/segments/GET/UserSegmentsList.json.example
+                            application/vnd.ibexa.api.UserSegmentList+xml:
+                                type: UserSegmentList
+                                example: !include examples/user/users/user_id/segments/GET/UserSegmentsList.xml.example
+                    404:
+                        description: If User does not exist.
+            post:
+                displayName: Assign Segments to User
+                description: ''
```

Comment:
```suggestion
                description: Assign User to one or more Segments.
```
If that's correct, namely that you can assign to multiple Segments at the same time?

Reply:
That is indeed possible and correct.